src/main/java/net/mcreator/element/types/VillagerTrade.java

```diff
@@ -64,17 +64,13 @@ public static class Entry {
 		}
 	}
 
-	public boolean hasWanderingTrade() {
+	public boolean hasVillagerTrade(boolean isNormal) {
```

Comment:
Wandering trades are special case, so I think this should be `wandering`, not `isNormal`.